tasks/task2.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const app = express();
+const fs = require('fs');
+
+const port = 3000;
+
+app.get('/', (req, res) => {
+  res.send('Greeting from Server');
+});
+
+app.get('/data', (req, res) => {
+  res.send('Here is some data');
+});
+
+app.listen(port, () => {
+  console.log(`Server is listening on port ${port}`);
+});
+
+fs.writeFile('./files/file.txt', '', (err) => {
+  if(err) throw err;
```

Comment:
This function should create file .txt , and write there numbers from 1 to 10.
But it just create empty file